db/migrate/20220321091220_add_personal_team_id_to_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddPersonalTeamIdToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :personal_team_id, :integer
+
+    User.all.find_each do |user|
```

Comment:
```suggestion
    User::Human.all.find_each do |user|
```
we don't want Api User to have a personal team